core/src/main/java/io/grpc/Status.java

```diff
@@ -315,6 +315,13 @@ public static Status fromCodeValue(int codeValue) {
   }
 
   /**
+   * Return a {@link Status} given a canonical error {@link Code} object.
+   */
+  public static Status fromCode(Code code) {
+    return fromCodeValue(code.value());
```

Comment:
This can just be code.status(), since even when status() is private, this method has access to it.